libuuu/usbhotplug.cpp

```diff
@@ -160,7 +160,7 @@ static int usb_add(libusb_device *dev)
 	if (item)
 	{
 		g_known_device_appeared = 1;
-		std::thread(run_usb_cmds, item, dev).detach();
+		std::thread(run_usb_cmds, item, dev).join();
 	}
```

Comment:
can't join here, which will block scan thread to find new boards.  If you plug two boards at the same, the second one have to wait for first one finish.  Lost parallel download feature.

Reply:
Ok, I see, wasn't aware of that feature. In that case using `detach()` is of course the right thing to do.

With properly linked static build I actually can use `detach()` successfully. So this can be closed.